docs/source/apps/cs2cs.rst

```diff
@@ -175,6 +175,14 @@ normally be in DMS format (use ``-f %.12f`` for decimal degrees with 12 decimal
 places), while projected (cartesian) coordinates will be in linear
 (meter, feet) units.
 
+Use of remote grids
```

Comment:
Perhaps this should also be added to `docs/source/apps/cct.rst`. I assume the same applies.

Reply:
done in https://github.com/OSGeo/PROJ/pull/1839/commits/4d21a242a10cf0e3c5b9503a9d0e8013ed4afd78